infra/src/main/resources/db/migration/repeatable/R__07.01_feature_type_data.sql

```diff
@@ -0,0 +1,64 @@
+-- Codes from https://wiki.buildingsmart.fi/fi/04_Julkaisut_ja_Standardit/infrabim_nimikkeisto
```

Comment:
Init-kama oli versioidussa migraatiossa, jotta alkuimportin IM:t saatiin sisään. Pistin päivityksen repeatableen jotta tätä listaa voi nyt korjailla in-place.